projects/laji/test/+save-observations/save-observations.po.ts

```diff
@@ -3,6 +3,7 @@ import { NavPage } from '../shared/nav.po';
 
 export class SaveObservationsPage {
 
+  _page = this.page;
```

Comment:
No need to have an extra variable, just set the constructor variable public (`constructor(public page: Page) { }`)